packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -304,28 +296,16 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     />
   );
 
-  useEffect(() => {
-    if (containerRef.current) {
-      containerRef.current.id = `video-gallery-${uuidv4()}`;
-    }
-  }, [containerRef]);
+  const horizontalGalleryPresent = horizontalGalleryTiles && horizontalGalleryTiles.length > 0;
 
   return (
     <div
       data-ui-id={ids.videoGallery}
       ref={containerRef}
       className={mergeStyles(videoGalleryOuterDivStyle, styles?.root)}
     >
-      {shouldFloatLocalVideo && (
-        <Modal
-          isOpen={true}
-          isModeless={true}
-          dragOptions={DRAG_OPTIONS}
-          styles={floatingLocalVideoModalStyle(theme, isNarrow)}
-          layerProps={{ hostId: containerRef.current?.id }}
-        >
-          {localParticipant && localVideoTile}
-        </Modal>
+      {shouldFloatLocalVideo && localParticipant && (
```

Comment:
nit: this is not 'float' anymore but more 'isolate' or 'separate'

Reply:
Good point. But I confirmed with Alex P and it should still be floating when there is no HorizontalGallery. So I updated the PR.

Reply:
agree floating is not ideal, lets land this PR and then maybe post in the WebUI channel if we want to change the enum and what we would change it to?